base/array.jl

```diff
@@ -919,6 +919,11 @@ function =={T<:BitInteger}(a::Array{T,1}, b::Array{T,1})
         :memcmp, Int32, (Ptr{T}, Ptr{T}, UInt), a, b, sizeof(T) * len)
 end
 
+"""
+    reverse(v, start=1, stop=length(v))
+
+Return a copy of `v` reversed from start to stop.
+"""
 function reverse(A::AbstractVector, s=1, n=length(A))
```

Comment:
Aside: the arg names do not match and it seems like 'stop' is incorrect terminology here...